src/app.jsx

```diff
@@ -278,12 +278,9 @@ class AppActive extends React.Component {
                     <>
                         {allNotifications}
                         <EmptyStatePanel title={ cockpit.format(_("VM $0 does not exist on $1 connection"), cockpit.location.options.name, cockpit.location.options.connection) }
-                                         action={
-                                             <Button variant="link"
-                                                     onClick={() => cockpit.location.go(["vms"])}>
-                                                 {_("Go to VMs list")}
-                                             </Button>
-                                         }
+                                         action={_("go to VMs list")}
```

Comment:
Please use the original string with a capital "G". This otherwise breaks all translations, and it also looks better IMHO.